src/System.ServiceModel.Security/tests/ServiceModel/SecutityBindingElementTest.cs

```diff
@@ -42,4 +42,26 @@ public static void Property_KeyEntropyMode()
         TransportSecurityBindingElement securityBindingElement = new TransportSecurityBindingElement();
         Assert.Equal(securityBindingElement.KeyEntropyMode, SecurityKeyEntropyMode.CombinedEntropy);
     }
+
+    [WcfFact]
+    public static void Method_CreateIssuedTokenOverTransportBindingElement()
+    {
+        IssuedSecurityTokenParameters tokenParameters = new IssuedSecurityTokenParameters();
+        TransportSecurityBindingElement securityBindingElement = SecurityBindingElement.CreateIssuedTokenOverTransportBindingElement(tokenParameters);
+
+        Assert.Contains(tokenParameters, securityBindingElement.EndpointSupportingTokenParameters.Endorsing);
+        Assert.Equal(System.ServiceModel.MessageSecurityVersion.Default, securityBindingElement.MessageSecurityVersion);
+        Assert.False(securityBindingElement.LocalClientSettings.DetectReplays);
+        Assert.True(securityBindingElement.IncludeTimestamp);
+
+        tokenParameters.KeyType = System.IdentityModel.Tokens.SecurityKeyType.BearerKey;
```

Comment:
We don't yet want to light up SecurityKeyType.BearerKey
In the setter for KeyType could you throw `throw ExceptionHelper.PlatformNotSupported();`
https://github.com/dotnet/wcf/blob/master/src/System.Private.ServiceModel/src/System/ServiceModel/Security/Tokens/IssuedSecurityTokenParameters.cs#L101

You can add it to this same PR and then remove the checks you `if (issuedTokenParameters.KeyType == SecurityKeyType.BearerKey)`

Reply:
Addressed.